src/main/java/com/kevin/tiertagger/mixin/AbstractClientPlayerEntityMixin.java

```diff
@@ -0,0 +1,19 @@
+package com.kevin.tiertagger.mixin;
+
+import com.kevin.tiertagger.TierTagger;
+import net.minecraft.client.network.AbstractClientPlayerEntity;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(AbstractClientPlayerEntity.class)
+public class AbstractClientPlayerEntityMixin {
+    @Inject(at = @At(value = "HEAD",
+            target = "Lnet/minecraft/client/network/AbstractClientPlayerEntity;tick()V",
+            ordinal = 0), method = "tick()V")
+    private void onTick(CallbackInfo ci)
+    {
+        TierTagger.hue = (float) (System.currentTimeMillis() % 3000) / 3000;
```

Comment:
just keep a tick counter in a long somewhere, you don't need to call `System#getCurrentTimeMillis` every tick